kokoro/linux/build.sh

```diff
@@ -81,6 +81,46 @@ do
 done
 echo $(date): Smoketests completed.
 
+set -e
+
 # Build the release packages.
 mkdir $BUILD_ROOT/out
 $SRC/kokoro/linux/package.sh $BUILD_ROOT/out
+
+##
+## Test capture and replay of the Vulkan Sample App.
+##
+
+# Install the Vulkan loader and xvfb (X virtual framebuffer).
+sudo apt-get -qy install libvulkan1 xvfb
+
+# Get prebuilt SwiftShader.
```

Comment:
Add a comment saying why we use this particular commit, or else to say that we just take the latest commit available and this should be updated from time to time.

Reply:
Done thanks. I also added a comment explaining that we have to disable the coherent memory tracker with SwiftShader.